client/my-sites/email/email-management/email-home.jsx

```diff
@@ -126,14 +125,13 @@ class EmailManagementHome extends Component {
 
 		return this.renderContentWithHeader(
 			<>
-				{ showActiveDomainList && (
-					<EmailListActive
-						currentRoute={ currentRoute }
-						domains={ domainsWithEmail }
-						selectedSiteId={ selectedSiteId }
-						selectedSiteSlug={ selectedSite.slug }
-					/>
-				) }
+				<EmailListActive
```

Comment:
This now needs to be rendered conditionally based on the `showActiveDomainList` prop